must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -140,6 +140,9 @@ for ns in $namespaces; do
         # Collecting rbd mirroring info for ceph rbd volumes
         printf "collecting rbd mirroring info for ceph rbd volumes \n" | tee -a  "${BASE_COLLECTION_PATH}"/gather-ceph-debug.log
         COMMAND_OUTPUT_FILE=${COMMAND_OUTPUT_DIR}/rbd_vol_and_mirror_info
+        # Checking snapshot schedule status
```

Comment:
Add two printf stating what we are doing here.
1 printf for COMMAND_OUTPUT_FILE
second printf for gather-ceph-debug-log file like: 
`printf "collecting snapshot schedule status \n" | tee -a  "${BASE_COLLECTION_PATH}"/gather-ceph-debug.log `